### DISCUSSION TITLE
GPU supported?

### DISCUSSION THREAD

**Discussion by Eiritj** [oobabooga/text-generation-webui#5673] [Q&A]
First of all I hope this is the right place to ask.
I'm running the text generation web UI as an add on in Home Assistant. Home Assistant is running on bare-metal, with a Ryzen 5 3600, 16Gb of RAM. Currently I'm not using any GPU, but after playing with the model "Home-3B-v3.q8_0.gguf" I was think to look at larger models, like the Mistral 7B. For that model I would need at least 14Gb of RAM, or VRAM, minimum. I was considering a RX 6900 XT since it has 16Gb of VRAM. Would that be be relatively plug and play? 
I see the RX 6900 XT is not on the official list of ROCm GPU's, but that it is listed on unofficial lists.
Any input on the expected success I can expect would be greatly appreciated.

**Comment by TheLounger** [comment]
You can technically use any card using Vulkan, going back to 2012 or so, but it will be slower than ROCm and CUDA.
Other than that, I would stay away from AMD unless it's a well-supported card, or you like to fiddle with getting it to work (again).
